packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -56,6 +56,9 @@ class LocalFileDestinationProvider implements IDestinationProvider {
 
     // Encryption
     if (this.options.encryption.enabled) {
+      if (!this.options.encryption.key) {
```

Comment:
Shouldn't we do this check during the bootstrap or even the constructor? So that we don't start the transfer if there is an issue.

Reply:
I added it to bootstrap (since I guess it's possible someone could want to instantiate a provider without setting the key until later) but I would also leave this check here in case it gets modified somehow between steps. And also because then typescript (rightfully) complains it could be null, essentially for that reason.

Reply:
Yep make sense, thanks